docs/LogGuide.md

```diff
@@ -1172,9 +1209,13 @@ although the fields before the `id` infrequently have non-zero values.
 Note: It's unclear when the head markers disappear.
 Maybe one of these fields is a duration time? It's not clear what either of these unknown values mean.
 
-Also, this appears to only be true on later fights.
-Turn 5 fireball and conflag headmarkers are actions from Twintania and not `NetworkTargetIcon` lines.
-It seems likely this was implemented later and nobody wanted to break old content by updating it to use newer types.
+Also, this appears to only be true on later fights. Turn 5 fireball and conflag headmarkers are actions from Twintania
+and not `NetworkTargetIcon` lines. It seems likely this was implemented later and nobody wanted to break old content by
+updating it to use newer types.
+
+Note that newer content uses 'sequential headmarkers' - every headmark is offset by a per-instance value. You will need
+to wait until you see the first headmarker in the instance, and then use this as an offset by which to adjust all the
+other IDs you see.
```

Comment:
Might be worth noting here that this is only the case for Extreme+ content, and thus it's relatively simple to get marker IDs from normal versions of encounters.

Reply:
I'm not sure this is even correct, the overhead icons are VFX animations from the LockOn EXD. Their values are references to the specific VFX. The duration is available, but you need to parse the actual AVFX file to get the duration. There's also a separate actor control event for explicit removals, but ACT doesn't parse it.